src/components/tabs/tabs.module.css

```diff
@@ -24,19 +29,24 @@ ref: https://twitter.com/heydonworks/status/408263890284191745
 ref: https://twitter.com/MarcoInEnglish/status/363284092806119424
 */
 .tabPanel[role='tabpanel'] {
+  /* Note: g-focus-ring-from-box-shadow added in JSX */
+  border-radius: 6px;
+
   &[aria-hidden='true'] {
     display: none;
     visibility: hidden;
   }
-}
 
-/*
-For showing a focus ring with a border radius when navigating via keyboard.
-*/
-.tabPanel {
-  composes: g-focus-ring-from-box-shadow from global;
+  &.allowNestedStyles:not(.isNested) {
+    box-shadow: var(--token-surface-base-box-shadow);
+    padding: 16px;
+  }
+
+  & > *:first-child {
+    margin-top: 0 !important;
```

Comment:
⛏️ nit: a note why we're using `!important` here would be nice!

Reply:
✅  Great call - added, & linked to 🎟️ [Convert global-ish docs content CSS to MDX custom components](https://app.asana.com/0/1201987349274776/1201653123138729/f)